pkg/processor/runtime/shell/runtime.go

```diff
@@ -99,7 +101,13 @@ func (s *shell) ProcessEvent(event nuclio.Event, functionLogger nuclio.Logger) (
 	s.Logger.DebugWith("Shell executed",
 		"eventID", event.GetID())
 
-	return out, nil
+	result := nuclio.Response{
```

Comment:
From other examples I've looked at in the code, seems like everywhere else we just do `return nuclio.Response{ ... }, nil` (with newlines still appearing)